openshift/clowder/clowd-app.yaml

```diff
@@ -13,6 +13,23 @@ objects:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: signing-script
+    namespace: automation-hub
+  data:
+    collection_sign.sh: |
+      IyEvYmluL2Jhc2gKCkZJTEVfUEFUSD0kMQpTSUdOQVRVUkVfUEFUSD0iJDEuYXNjIgoKQURNSU5fSUQ9ImdhbGF4eTNAYW5zaWJsZS5jb20iClBBU1NXT1JEPSJHYWxheHkyMDIyIgoKIyBDcmVhdGUgYSBkZXRhY2hlZCBzaWduYXR1cmUKZ3BnIC0tcXVpZXQgLS1iYXRjaCAtLXBpbmVudHJ5LW1vZGUgbG9vcGJhY2sgLS15ZXMgLS1wYXNzcGhyYXNlICAgJFBBU1NXT1JEIC0taG9tZWRpciAvdG1wL2Fuc2libGUvLmdudXBnIC0tZGV0YWNoLXNpZ24gLS1kZWZhdWx0LWtleSAkQURNSU5fSUQgICAtLWFybW9yIC0tb3V0cHV0ICRTSUdOQVRVUkVfUEFUSCAkRklMRV9QQVRICgojIENoZWNrIHRoZSBleGl0IHN0YXR1cwpTVEFUVVM9JD8KaWYgWyAkU1RBVFVTIC1lcSAwIF07IHRoZW4KICBlY2hvIHtcImZpbGVcIjogXCIkRklMRV9QQVRIXCIsIFwic2lnbmF0dXJlXCI6IFwiJFNJR05BVFVSRV9QQVRIXCJ9CmVsc2UKICBleGl0ICRTVEFUVVMKZmkK
```

Comment:
@rochacbruno so for clowder will we store the bash script in our cloud env as a secret and reference it via this hash?

Reply:
Secrets defined in the clowd-app.yaml are only used in ephemeral. For stage/prod cloud environments, the secret will be populated from Vault.

Reply:
@drodowic @awcrosby In fact, there will be no SigningService on cloud for now, but the functionality will still be there in case we add this in the future, as cloud will be upload-only this script is not going to be used.

New plan is to deploy only the public key to cloud and use pulp_ansible upload to verify the uploaded signature.

So for now the changes impacts only Ephemeral environments (which I assume is also where QE will test some of the features)